src/components/ui/UserProfile.js

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import "styles/views/UserProfile.scss"
 import "styles/ui/Button.scss"
-import {Button} from "../ui/Button";
+import {Button} from "./Button";
```

Comment:
why take the button out of the ui directory?

Reply:
I didn't take the button out of the ui, but put UserProfile into ui ;)